packages/core/src/external/aws/lambda-logic/bundle-to-html-adhd.ts

```diff
@@ -0,0 +1,2339 @@
+          }
+
+          table {
+            width: 100%;
+            margin: 0.3em 0;
+          }
+
+          tbody tr, tbody th {
+            background-color: #f2f2f2;
+          }
+
+          tbody tr td {
+            padding: 8px 5px;
+          }
+
+          .section {
+            margin-bottom: 50px;
+          }
+
+          .section-title {
+            display: flex;
+            align-items: center;
+            justify-content: space-between;
+          }
+
+          .section-title a {
+            text-decoration: none;
+            color: black;
+          }
+
+          .span_button {
+            display: table-cell;
+            cursor: pointer;
+            border: 1pt inset #585858;
+            border-radius: 5px;
+            -moz-border-radius: 15px;
+            padding: 0.2cm 0.4cm;
+            background-color: #f2f2f2;
+            font-weight: bold;
+            vertical-align: baseline;
+          }
+
+          #nav {
+            border: 1px solid;
+            border-radius: 5px;
+            padding: 20px;
+            margin: 0;
+            background-color: #f2f2f2;
+            display: flex;
+            justify-content: space-between;
+          }
+
+
+          #nav .half {
+            width: 50%;
+          }
+
+          #nav li {
+            margin-bottom: 10px;
+            margin-left: 20px;
+          }
+
+          #nav li a {
+            text-decoration: none;
+            color: black;
+          }
+
+          #report {
+            border: 1px solid;
+            margin-bottom: 20px;
+            border-radius: 5px;
+            padding: 20px;
+          }
+
+          #report .header {
+            display: flex;
+            justify-content: space-between;
+          }
+
+          #report .header .title {
+            margin: 0;
+          }
+
+          #report .labs a {
+            text-decoration: none;
+            color: black;
+          }
+
+          .documentation .divider {
+            display: none;
+          }
+
+          .reason-for-visit .divider {
+            display: none;
+          }
+
+          #mr-header h4 {
+            margin-bottom: 10px
+          }
+
+        </style>
+      </head>
+
+      <body>
+        ${createMRHeader(patient)}
+        <div class="divider"></div>
+        <div id="mr-sections">
+          ${createFilteredReportSection(
+            diagnosticReports,
+            practitioners,
+            aweVisits,
+            organizations,
+            "Annual Wellness Exams",
+            "awe",
+            aYearAgo
+          )}
+          ${createDiagnosticReportsSection(
+            diagnosticReports,
+            practitioners,
+            [...aweVisits, ...adhdVisits],
+            organizations
+          )}
+          ${createFilteredReportSection(
+            diagnosticReports,
+            practitioners,
+            adhdVisits,
+            organizations,
+            "ADHD",
+            "adhd"
+          )}
```

Comment:
new `createFilteredReportSection` which takes the conditions we want to filter for and also some text to build out the section